Tools/Scripts/run-javascriptcore-tests

```diff
@@ -46,6 +46,9 @@ use webkitdirs;
 use Scalar::Util qw(looks_like_number);
 use Text::ParseWords;
 
+sub trim { my $s = shift; $s =~ s/^\s+|\s+$//g; return $s };
```

Comment:
We always use `chomp` for this, no need to introduce a new function.

Reply:
Fixed.